exotic/api/ephemeris.py

```diff
@@ -1369,6 +1384,7 @@ def prior_transform(upars):
 
     # TODO extend plot, BIC Values
 
+    # should this be here?
     dude()
```

Comment:
@pearsonkyle Yes, I read this far down. Is this a placeholder for something else? Should we remove it?

Reply:
Cleaned up